src/dbos.ts

```diff
@@ -226,13 +226,27 @@ export class DBOS {
     // Initialize the DBOS executor
     if (!DBOS.dbosConfig) {
       const [dbosConfig, runtimeConfig] = parseConfigFile({ forceConsole: isDebugging });
-      if (!isDebugging) {
+      if (!isDebugging && dbosConfig.poolConfig) {
```

Comment:
In what circumstance could there not be a poolConfig here?

Reply:
never. I'll just `!`